CMakeLists.txt

```diff
@@ -9,17 +9,13 @@
 
 cmake_minimum_required( VERSION 3.12 )
 
-set(iodaconv_major 0)
-set(iodaconv_minor 0)
-set(iodaconv_revision 1)
-set(iodaconv_version ${iodaconv_major}.${iodaconv_minor}.${iodaconv_revision})
-project( iodaconv VERSION ${iodaconv_version} LANGUAGES Fortran )
+project( iodaconv VERSION 0.0.1 LANGUAGES C CXX Fortran )
 
 ## Configuration
 include(GNUInstallDirs)
 include(CTest)
 
-set( CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake;${CMAKE_MODULE_PATH} )
+list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
```

Comment:
This changes the order of the module search path.  I would either stick with the original code or do `list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_CURRENT_SOURCE_DIR}/cmake )`

Reply:
Ok.